src/manager/frontend/src/components/sections/services.tsx

```diff
@@ -102,7 +119,7 @@ function StartStopButton({
     >
       <Pause size={20} />
     </Button>
-  ) : (
+  ) : service.status === "exited" ? (
```

Comment:
when do we show status exited or missing in the UI

Reply:
exited:
-> the container is exist but it's not running (admin pressed stop button or it's stopped for some reason)

missing:
-> the manager has the app config but there is no container of it. Ex, you ran `docker container rm <container_name>` to remove the container directly and not through the manager. This only occur after the manager has been started because we already have a task to start the missing app (and remove orphaned container as well)

Reply:
the manager role is a non technical user - we should make it as simple as possible

Reply:
still showing exited and missing?